src/Cli/InstallCommand.php

```diff
@@ -10,6 +10,14 @@
 
 class InstallCommand extends Command
 {
+    private $dockerInstaller;
+
+    public function __construct(DockerInstaller $dockerInstaller)
+    {
+        $this->dockerInstaller = $dockerInstaller;
+        parent::__construct();
```

Comment:
Strange ! If we call `setName` in the `configure` method, it shouldn't be a problem at all..